pkg/apis/componentconfig/types.go

```diff
@@ -397,6 +397,9 @@ type KubeletConfiguration struct {
 	// Currently only cpu and memory are supported. [default=none]
 	// See http://releases.k8s.io/HEAD/docs/user-guide/compute-resources.md for more detail.
 	KubeReserved utilconfig.ConfigurationMap `json:"kubeReserved"`
+	// apiServerList is a list of Kubernetes API servers for publishing events
+	// and reading pods and services. (ip:port), comma separated.
+	APIServerList []string `json:"apiServerList"`
```

Comment:
I thought we didn't add this to component config because this is information that should be in a kubeconfig file provided to the apiserver (as a client)?